web/main.py

```diff
@@ -4,7 +4,7 @@
 app = Flask(__name__)
 
 app.config["SQLALCHEMY_DATABASE_URI"] = 'mysql+pymysql://{usr}:{passwd}@{host}/{db}'.format(
-    usr='sneezy', passwd='password', host='db', db='immortal')
+    usr='sneezy', passwd='password', host='db', db='sneezy')
```

Comment:
OK, I tried adding another sneezy instance + db to docker-compose.  Each instance talks to it's own db just fine, but the issue I'm having is with the lib files getting shared by the two sneezy instances due to the way the whole directory is mounted.  It would be simple just to add another copy of the sneezymud files for the builder to work in, but this would make the git workflow more complicated for sneezy updates.  Although in that scenario we could also beta test sneezy code on the builder port.